guides/1)getting_started/1)quickstart.md

```diff
@@ -1,110 +1,108 @@
+**Prerequisite**: Gradio requires Python 3.7 or higher, that's all!
 
-## What does Gradio Do?
+## What Does Gradio Do?
 
-One of the *best ways to share* your machine learning model, API, or data science workflow with others is to create an **interactive app** that allows your users or colleagues to try out the demo in their browsers. 
+One of the *best ways to share* your machine learning model, API, or data science workflow with others is to create an **interactive app** that allows your users or colleagues to try out the demo in their browsers.
 
-Gradio allows you to **build demos and share them, all in Python.** And usually in just a few lines of code! So let's get started. 
+Gradio allows you to **build demos and share them, all in Python.** And usually in just a few lines of code! So let's get started.
 
 ## Hello, World
 
 To get Gradio running with a simple "Hello, World" example, follow these three steps:
 
-<span>1.</span> Install Gradio from pip. Note, the minimal supported Python version is 3.7.
+1\. Install Gradio using PIP:
 
 ```bash
 pip install gradio
 ```
 
-<span>2.</span> Run the code below as a Python script or in a Python notebook (or in a  [colab notebook](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing)).
+2\. Run the code below as a Python script or in a Jupyter Notebook ([Google Colab](https://colab.research.google.com/drive/18ODkJvyxHutTN0P5APWyGFO_xwNcgHDZ?usp=sharing) also is a good option):
 
 $code_hello_world
 
-<span>3.</span> The demo below will appear automatically within the Python notebook, or pop in a browser on  [http://localhost:7860](http://localhost:7860/)  if running from a script.
+3\. The demo below will appear automatically within the Jupyter Notebook, or pop in a browser on [http://localhost:7860](http://localhost:7860) if running from a script:
 
 $demo_hello_world
 
 ## The `Interface` Class
 
-You'll notice that in order to make the demo, we created a `gradio.Interface`. This `Interface` class can wrap any Python function with a user interface. In the example above, we saw a simple text-based function. But the function could be anything from music generator to a tax calculator to the prediction function of a pretrained machine learning model.
+You'll notice that in order to make the demo, we created a `gradio.Interface`. This `Interface` class can wrap any Python function with a user interface. In the example above, we saw a simple text-based function, but the function could be anything from music generator to a tax calculator to the prediction function of a pretrained machine learning model.
 
 The core `Interface` class is initialized with three required parameters:
 
--   `fn`: the function to wrap a UI around
--   `inputs`: which component(s) to use for the input, e.g. `"text"` or `"image"` or `"audio"` 
--   `outputs`: which component(s) to use for the output, e.g. `"text"` or `"image"` `"label"`
+- `fn`: The function to wrap a UI around.
```

Comment:
Let's stick to lower case and without the period since these are not full sentences